app/views/transactions/index.html.slim

```diff
@@ -1,3 +1,5 @@
 h2
   | All transactions
 = render 'transactions/transactions'
+
+= will_paginate(@transactions, renderer: WillPaginate::ActionView::Bootstrap4LinkRenderer)
```

Comment:
@marderer can't we configure `renderer` in will_paginate initializer? In order not to go with `WillPaginate::ActionView::Bootstrap4LinkRenderer` in views, looks strange